students/david_russo/lesson2/series.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+def fibonacci(n):
+    """
+    The function fibonacci returns the nth term of the Fibonacci sequence. The first two elements of 
+    the fibonacci sequence are initialized with values of 0 and 1. Then, subsequent values are created by
+    adding the previous two values of the sequence. The third element is equal to the sum of the 1st
+    and 2nd elements (i.e., 0 + 1 = 1), the fourth element is the sum of the 2nd and 3rd elements, 
+    and so on and so forth. As you can see, the sequence is constructed recursively. 
+    """
+    if n == 1:
+        return 0
+    elif n == 2:
+        return 1
+    else:
+        results = []
+        results.append(0)
+        results.append(1)
```

Comment:
You can instantiate `results = [0, 1]` without having the call the first two `.append` statements.